src/zap/zapcode.cpp

```diff
@@ -1202,6 +1202,30 @@ void ZapUnwindData::Save(ZapWriter * pZapWriter)
 #endif //REDHAWK
 }
 
+#elif defined(_TARGET_X86_) && defined(FEATURE_PAL)
+
+UINT ZapUnwindData::GetAlignment()
+{
+    return sizeof(BYTE);
```

Comment:
This change was introduced according to the following comment by @jkotas:
>Can this be just sizeof(BYTE) ? The JIT case does not seem to have any extra alignment. It can be same here.

Could you let us know the semantics of ``GetAlignment``? x86/Linux currently emits no unwind data, and thus ``sizeof(BYTE)`` also looks fine.